infra/src/test/scala/fr/gospeak/infra/services/storage/sql/testingutils/RepoSpec.scala

```diff
@@ -38,20 +38,22 @@ class RepoSpec extends FunSpec with Matchers with IOChecker with BeforeAndAfterE
   protected val venue: Venue = random[Venue]
   protected val sponsorPack: SponsorPack = random[SponsorPack]
   protected val sponsor: Sponsor = random[Sponsor]
-  protected val params = Page.Params()
+  protected val params: Page.Params = Page.Params()
```

Comment:
Seems redumdant with `page` attr, could be removed

Reply:
Done